<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purchase_order_widget</class>
 <widget class="QWidget" name="purchase_order_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Purchase Order</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>604</width>
     <height>926</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="20" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Confirmed Delivery To:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="is_partial_delivery">
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="delivery_address_postcode">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Tax Percentage:</string>
        </property>
       </widget>
      </item>
      <item row="19" column="3">
       <widget class="QLineEdit" name="confirmed_delivery_from">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="delivery_address_name">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Line Number:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Phone:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label6">
        <property name="text">
         <string>Tax Amount:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>Requested Delivery To:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLineEdit" name="delivery_address_country">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="delivery_address_city">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>Gross Amount:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="acknowledged_at">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label8">
        <property name="text">
         <string>Acknowledged At:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Lines:</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Units: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="purchase_order_id">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Item Number: </string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>Requested Delivery From:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="17" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Gross Amount: </string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QLineEdit" name="confirmed_delivery_to">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Country</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Gross Price: </string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="item_no">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>Created At:</string>
        </property>
       </widget>
      </item>
      <item row="16" column="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Quantity Recieved:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>City</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Quantity Accepted:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Purchase Order Details:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Supplier Item Number:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Quantity Dispatched:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="delivery_address_address">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="delivery_address_email">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Purchase Order ID:</string>
        </property>
       </widget>
      </item>
      <item row="19" column="2">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Confirmed Delivery From:</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QLineEdit" name="units">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label9">
        <property name="text">
         <string>Is Partial Delivery:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="created_at">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="requested_delivery_to">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label7">
        <property name="text">
         <string>Total Amount: </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="gross_amount"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Delivery Address:</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Tax Amount: </string>
        </property>
       </widget>
      </item>
      <item row="18" column="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Total Amount: </string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Postcode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="delivery_address_phone">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="supplier_item_no">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="requested_delivery_from">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="total_amount"/>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="tax_amount"/>
      </item>
      <item row="13" column="3">
       <widget class="QSpinBox" name="lines_line_number"/>
      </item>
      <item row="15" column="1">
       <widget class="QSpinBox" name="lines_quantity"/>
      </item>
      <item row="16" column="3">
       <widget class="QSpinBox" name="lines_quantity_received"/>
      </item>
      <item row="15" column="3">
       <widget class="QSpinBox" name="lines_quantity_dispatched"/>
      </item>
      <item row="14" column="3">
       <widget class="QSpinBox" name="lines_quantity_accepted"/>
      </item>
      <item row="18" column="1">
       <widget class="QSpinBox" name="lines_tax_percentage"/>
      </item>
      <item row="17" column="1">
       <widget class="QDoubleSpinBox" name="lines_gross_price"/>
      </item>
      <item row="19" column="1">
       <widget class="QDoubleSpinBox" name="lines_tax_amount"/>
      </item>
      <item row="17" column="3">
       <widget class="QDoubleSpinBox" name="lines_gross_amount"/>
      </item>
      <item row="18" column="3">
       <widget class="QDoubleSpinBox" name="lines_total_amount"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="styleSheet">
         <string notr="true">min-height: 1.5em;</string>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
